cli/commands.go

```diff
@@ -28,6 +28,17 @@ const (
 	defaultServerCount = 1
 )
 
+var defaultBindMounts = make([]string, 0)
+
+func init() {
```

Comment:
Maybe we can rather have a `checkDefaultBindMounts` function in `util.go`, which takes a slice of default mounts (from a const) and returns a slice of bindMounts to `createServer`?